bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx127-eu.html

```diff
@@ -0,0 +1,153 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros.html" import video_inline_embed with context %}
+{% from "macros-protocol.html" import picto with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_whatsnew_127_eu') }}
+{% endblock %}
+
+{% if LANG == 'fr' %}
+  {% set main_title = '1:0 pour l’équipe multitâche' %}
+  {% set tagline_one = 'Profitez de ce summer of sports avec l’incrustation vidéo de Firefox, et regardez les matchs, ou courses, pendant que vous travaillez.' %}
+  {% set tagline_two = 'Testez la fonctionnalité avec la vidéo ci-dessus. Appuyez sur “lecture” et cliquez sur <strong>le bouton Incrustation vidéo</strong> pour faire sortir la vidéo.'|safe %}
+  {% set cta = 'Choisir Firefox comme navigateur par défaut' %}
+  {% set sub_title = 'Idées d’utilisation de l’incrustation vidéo' %}
+  {% set item_lecture = 'Assistez à une réunion tout en prenant des notes' %}
+  {% set item_cook = 'Gardez un tuto ouvert au-dessus d’une recette' %}
+  {% set item_entertain = 'Divertissez les enfants pendant que vous travaillez' %}
+{% elif LANG == 'de' %}
+  {% set main_title = 'Das Eckige muss ins … andere Eckige?!' %}
+  {% set tagline_one = 'Mit dem „Bild-im-Bild“-Feature von Firefox kannst du Fußball gucken und gleichzeitig arbeiten.' %}
+  {% set tagline_two = 'Probier es direkt mit dem Video über diesem Text. Drück auf play und anschließend auf <strong>den Bild-im-Bild-Button</strong>, um das Video zu lösen.'|safe %}
+  {% set cta = 'Firefox als Standardbrowser festlegen' %}
+  {% set sub_title = 'Bild-im-Bild machts möglich' %}
+  {% set item_lecture = 'Ein Meeting verfolgen und gleichzeitig Notizen machen' %}
+  {% set item_cook = 'Ein Koch-Tutorial angucken, während du das Rezept anschaust' %}
+  {% set item_entertain = 'Die Kinder entertainen und nebenbei eine E-Mail fertig schreiben' %}
+{% elif LANG == 'es-ES' %}
+  {% set main_title = 'Multitarea: 1 <br>FOMO: 0'|safe %}
```

Comment:
**Suggestion:** would it be worth explaining what the abbreviation FOMO stands for?
```suggestion
  {% set main_title = 'Multitarea: 1 <br><abbr title="Fear of missing out">FOMO</abbr>: 0'|safe %}
```

Same for other occurrences below?